### ISSUE TITLE
missing endpoints to bulk delete multiple collections/groups

### ISSUE THREAD

**Issue by stefan0xC** [dani-garcia/vaultwarden#3350]
### Subject of the issue
While testing the latest `testing`image (vaultwarden v1.27.0-729b5631 / web-vault v2023.2.0), I noticed that with the new consolidated view it is now possible to bulk delete collections and groups in the organization tab, however this does not currently work because the respective endpoints are missing.

### Steps to reproduce

Create an organization and add two empty collections (if you have set `ORG_GROUPS_ENABLED=true` you can test the same with groups). Select both collections and click on `Delete selected` in the uppermost three-dot menu and confirm.

[Peek 2023-03-16 20-38.webm](https://user-images.githubusercontent.com/509385/225745695-30a5d4ec-9bce-4f37-bd10-71919a6c590b.webm)

### Troubleshooting data

The XHR request to `DELETE <DOMAIN>/api/organizations/85435556-697e-4441-91b6-14486c9469c6/collections` with data:
```json
{"ids":["95bfdea9-70e7-4ee5-956d-ecd25f53a9d3","3d652f44-cd05-4772-8d82-c8df8e89f6d3"],"organizationId":"85435556-697e-4441-91b6-14486c9469c6"}
```

The request to `DELETE <DOMAIN>/api/organizations/85435556-697e-4441-91b6-14486c9469c6/groups`:
```json
{"ids":["a99a1ede-bf09-44f3-8a39-9f5a2fd22d48","cf3b45fb-f553-4cb5-a437-0e89cd0f2e79"]}
```

As both endpoints are missing, they return the following response:
```json
{"error":{"code":404,"description":"The requested resource could not be found.","reason":"Not Found"}}
```

**Comment by BlackDex** [comment]
Good catch!

Now.... Have you fixed it already 🙈

**Comment by stefan0xC** [comment]
Nope, but I can try to fix it this weekend. I have no time to spend time on it earlier. :smiling_face_with_tear:
